pkg/controllers/updaterun/initialization.go

```diff
@@ -518,3 +524,14 @@ func (r *Reconciler) recordInitializationFailed(ctx context.Context, updateRun *
 	}
 	return nil
 }
+
+func (r *Reconciler) removeWaitTimeFromAfterStageTasks(ctx context.Context, updateStrategy *placementv1beta1.ClusterStagedUpdateStrategy) error {
+	for i := range updateStrategy.Spec.Stages {
+		for j := range updateStrategy.Spec.Stages[i].AfterStageTasks {
+			if updateStrategy.Spec.Stages[i].AfterStageTasks[j].Type == placementv1beta1.AfterStageTaskTypeApproval {
+				updateStrategy.Spec.Stages[i].AfterStageTasks[j].WaitTime = nil
+			}
+		}
+	}
+	return r.Update(ctx, updateStrategy)
```

Comment:
Let's use `r.Client.Update` to be consistent, also please wrap the error with `controller.NewUpdateIgnoreConflictError()`